examples/metrics/prometheus-install/strimzi-service-monitor.yaml

```
@@ -6,8 +6,8 @@ metadata:
     app: strimzi
 spec:
   selector:
-    matchLabels:
-      strimzi.io/kind: Kafka
+    matchExpressions:
+      - {key: "strimzi.io/kind", operator: In, values: ["Kafka", "KafkaConnect"]}
```

Does it work when it is instead written as YAML like the rest of the file?

```suggestion
      - key: strimzi.io/kind
        operator: In
        values: 
          - Kafka
          - KafkaConnect
```

---

Yes, it works in YAML form. I went ahead and changed it.